Quickstart example fails on `brig run`

Working through the Quickstart docs, everything works fine up until `brig run -f brigade.js deis/empty-testbed`.

- Brigade server is successfully installed from helm chart
- Deployed `brigade-project` out of the `charts/` directory with default `values.yaml`
- copy/pasted `brigade.js` contents from Quickstart guide
- when running `brig run -f brigade.js`, 

```
brig --verbose run -f ./brigade.js deis/empty-testbed
panic: runtime error: slice bounds out of range

goroutine 1 [running]:
github.com/Azure/brigade/brig/cmd/brig/commands.(*scriptRunner).send(0xc4202fd140, 0x7ffeefbff683, 0x12, 0xc420158a00, 0x7a, 0x27a, 0x0, 0x1ce36a2)
	/Users/alex/go/src/github.com/Azure/brigade/brig/cmd/brig/commands/run.go:132 +0x446
github.com/Azure/brigade/brig/cmd/brig/commands.glob..func5(0x23fc700, 0xc4203dbc80, 0x1, 0x4, 0x0, 0x0)
	/Users/alex/go/src/github.com/Azure/brigade/brig/cmd/brig/commands/run.go:84 +0xe5
github.com/Azure/brigade/vendor/github.com/spf13/cobra.(*Command).execute(0x23fc700, 0xc4203dbb80, 0x4, 0x4, 0x23fc700, 0xc4203dbb80)
	/Users/alex/go/src/github.com/Azure/brigade/vendor/github.com/spf13/cobra/command.go:599 +0x3e9
github.com/Azure/brigade/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x23fb820, 0xc420072058, 0x0, 0x1)
	/Users/alex/go/src/github.com/Azure/brigade/vendor/github.com/spf13/cobra/command.go:689 +0x339
github.com/Azure/brigade/vendor/github.com/spf13/cobra.(*Command).Execute(0x23fb820, 0x23fb820, 0xc4204dff50)
	/Users/alex/go/src/github.com/Azure/brigade/vendor/github.com/spf13/cobra/command.go:648 +0x2b
main.main()
	/Users/alex/go/src/github.com/Azure/brigade/brig/cmd/brig/main.go:11 +0x31
```

--- a/brig/cmd/brig/commands/run.go
+++ b/brig/cmd/brig/commands/run.go
@@ -129,7 +129,11 @@ func (a *scriptRunner) send(projectName string, data []byte) error {
 		return err
 	}
 
-	podName := fmt.Sprintf("brigade-worker-%s-%s", b.ID, b.Commit[0:8])
+	shortRef := b.Commit
+	if len(shortRef) > 8 {
+		shortRef = shortRef[0:8]
+	}
+	podName := fmt.Sprintf("brigade-worker-%s-%s", b.ID, shortRef)
 
 	if err := a.waitForWorker(b.ID); err != nil {
 		return err

